native_client/ctcdecode/ctc_beam_search_decoder.cpp

```diff
@@ -226,6 +226,7 @@ std::vector<Output> ctc_beam_search_decoder(
     std::shared_ptr<Scorer> ext_scorer,
     size_t num_results)
 {
+  VALID_CHECK_EQ(alphabet.GetSize()+1, class_dim, "wrong class_dim or wrong alphabet");
```

Comment:
This is good, thanks. Could you just expand the message a bit to be clearer on what's happening? Something like:

```suggestion
  VALID_CHECK_EQ(alphabet.GetSize()+1, class_dim, "Number of output classes in acoustic model does not match number of labels in the alphabet file. Alphabet file must be the same one that was used to train the acoustic model.");
```

Reply:
Ok, done. I have squashed my commit to avoid having useless commits.